test/unittests/state_transition_eof_create_test.cpp

```diff
@@ -1162,7 +1062,7 @@ TEST_F(state_transition, txcreate_initcontainer_return)
     tx.to = To;
     pre.insert(*tx.to, {.nonce = 1, .code = factory_container});
 
-    expect.post[*tx.to].nonce = pre.get(*tx.to).nonce + 1;
+    expect.post[*tx.to].nonce = pre.get(*tx.to).nonce;
     expect.post[*tx.to].storage[0x00_bytes32] = 0x00_bytes32;
```

Comment:
this expectation is void. If we want to keep it, we should set the storage on tx.to to non-zero value.

Same comment next test

Reply:
It's redundant, it's just for documentation to make explicit what's the outcome.

Reply:
It is not exactly void, the check is only done if a value is assigned to "expect". There are `std::optional`s.

Reply:
So as I understand it is good to have it here.

Reply:
Yes.